sdk/monitor/Azure.Monitor.Query/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
+## 1.1.0 (2021-12-07)
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `LogsTableRow` to `MonitorQueryModelFactory`
+- Added `LogsTable` to `MonitorQueryModelFactory`
```

Comment:
In addition to my suggestion below, can you add a sentence explaining why the user would care about this change? Also, are there any samples that need to be updated as a result of this change?

```suggestion

- Added `LogsTable` and `LogsTableRow` to `MonitorQueryModelFactory`
```